src/Writer/SQLTransformer.php

```diff
@@ -26,7 +26,7 @@ private static function transformColumn(array $column): string
                 $size = 'MAX';
             }
         }
-        if (self::isTextType($column['type'])) {
+        if (self::isTextType($column['type']) || self::isBinaryType($column['type'])) {
```

Comment:
Pri binarnych typoch musi byt v staging tabulke `NVARCHAR(MAX)` neviem presne preco, ale bez toho to neslo.